lib/master.js

```diff
@@ -214,7 +225,8 @@ Master.prototype._startWorkers = function(remainingWorkers) {
                     case 'startup_finished':
                         worker.removeListener('exit', workerExit);
                         self._currentStartingWorker = undefined;
-                        resolve(self._startWorkers(--remainingWorkers));
+                        res.push(msg.serviceReturns[0]);
```

Comment:
Why only the first element? As I understand the format of the return value could be
```
[ [ worker1.service1, worker1.service2 ], [worker2.service1, worker2.service2] ]
```